bot/internal/review/review.go

```diff
@@ -109,6 +109,8 @@ type Config struct {
 	// CodeReviewers and CodeReviewersOmit is a map of code reviews and code
 	// reviewers to omit.
 	CodeReviewers     map[string]Reviewer `json:"codeReviewers"`
+	CoreReviewers     map[string]Reviewer `json:"coreReviewers"`
+	CloudReviewers    map[string]Reviewer `json:"cloudReviewers"`
```

Comment:
nit: keep `CodeReviewers` and `CodeReviewersOmit` together so the comment above still applies to both of them.

(then you can add a separate comment that describes how `CoreReviewers` and `CloudReviewers` work)